.github/workflows/ci.yaml

```diff
@@ -10,20 +10,15 @@ on:
 
 
 jobs:
-  CI:
-    runs-on: ubuntu-18.04
-    timeout-minutes: 180
-    strategy:
-      fail-fast: true
+  test:
```

Comment:
I think this rename could make use unable to merge. ASF Infra group sets up `CI` as required task in default.
We could change the settings through [this file](https://github.com/apache/skywalking-rust/blob/master/.asf.yaml).